### PR TITLE
Add ansible/python/ruby linters

### PR DESCRIPTION
**Pull request by asciible**

### CODE REVIEW

**Review by cicharka — DISMISSED** [review]

File: docker-image/Dockerfile [hunk 1]
```diff
@@ -41,12 +41,24 @@ RUN apt-get update \
         docker-ce-cli \
         python3.7 \
         python3-pip \
+        rubygems \
 && apt-get clean \
 \
 && python3.7 -m pip install --upgrade \
-        pip setuptools wheel \
+        pip setuptools wheel lxml \
```
Comment (to-bar) [inline docker-image/Dockerfile]:
```suggestion
        pip setuptools wheel \
```

File: docker-image/Dockerfile [hunk 2]
```diff
@@ -41,12 +41,24 @@ RUN apt-get update \
...
-        boto3 jinja2 pytest python-json-logger ruamel.yaml
+        boto3 \
+        jinja2 \
+        pytest \
+        python-json-logger \
+        ruamel.yaml \
+        pylint \
+        pylint-fail-under \
+        ansible-lint \
+        ansible-lint-junit==0.16 \
```
Comment (to-bar) [inline docker-image/Dockerfile]: Please add a comment why we use 0.16 or remove fixed version

**Review by to-bar — CHANGES_REQUESTED** [review]

File: docker-image/Dockerfile [hunk 3]
```diff
@@ -13,6 +13,7 @@ SHELL ["/bin/bash", "-c"]
 # ppa:deadsnakes/ppa repo contains more recent Python versions packaged for Ubuntu
 # curl and gnupg-agent required to add Docker repo
 # software-properties-common provides add-apt-repository
+# ansible-lint-junit==0.16 version specified because default version installed by pip fails to execute
```
Comment (to-bar) [inline docker-image/Dockerfile]:
```suggestion
# lxml is required to build ansible-lint-junit wheel
```

File: docker-image/Dockerfile [hunk 4]
```diff
@@ -41,12 +42,24 @@ RUN apt-get update \
...
-        boto3 jinja2 pytest python-json-logger ruamel.yaml
+        boto3 \
+        jinja2 \
+        pytest \
+        python-json-logger \
+        ruamel.yaml \
+        pylint \
+        pylint-fail-under \
+        ansible-lint \
+        ansible-lint-junit==0.16 \
```
Comment (to-bar) [inline docker-image/Dockerfile]:
```suggestion
        ansible-lint \
        ansible-lint-junit>=0.16 \
        boto3 \
        jinja2 \
        pylint \
        pylint-fail-under \
        pytest \
        python-json-logger \
        ruamel.yaml \
```
Let's keep alphabetical order. The latest ansible-lint-junit in PyPI is 0.16 so I'm not sure if we need to set it. If yes, maybe >=0.16 would be better?

File: docker-image/Dockerfile [hunk 5]
```diff
@@ -41,12 +42,24 @@ RUN apt-get update \
         docker-ce-cli \
         python3.7 \
         python3-pip \
+        rubygems \
 && apt-get clean \
 \
 && python3.7 -m pip install --upgrade \
         pip setuptools wheel \
```
Comment (to-bar) [inline docker-image/Dockerfile]:
```suggestion
        lxml pip setuptools wheel \
```
It turned out that lxml is required (I've built image locally with `--no-cache`).

**Review by to-bar — CHANGES_REQUESTED** [review]

**Review by to-bar — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Pull request merged**